waiter/src/waiter/scheduler/kubernetes.clj

```diff
@@ -923,7 +923,29 @@
            :resources {:limits {:memory memory}
                        :requests {:cpu cpu :memory memory}}
            :volumeMounts [{:mountPath "/srv/www"
-                           :name "user-home"}]})))))
+                           :name "user-home"}]}))
+
+      ;; Optional envoy sidecar container
+      has-reverse-proxy?
+      (update-in
+        [:spec :template :spec :container]
+        conj
+        (let [cmd ["/bin/sh" "-x"]]
+             {:command cmd
+              :args ["./opt/waiter/envoy/bin/run-envoy-sidecar.sh"
+                     "$(SERVICE_PORT)"
+                     "$(PORT0)"]
```

Comment:
I still have to refactor this so that the whole setup would be just command "/bin/run-envoy" I'm using this :args setup currently since this is one of the few configurations that works.

Reply:
Fixed